source/config/configSpec.py

```diff
@@ -76,6 +76,7 @@
 	readByParagraph = boolean(default=false)
 	wordWrap = boolean(default=true)
 	focusContextPresentation = option("changedContext", "fill", "scroll", default="changedContext")
+	interruptSpeechWhileScrolling = boolean(default=true)

```

Comment:
Should this instead use the new feature flag API? @feerrenrut

Reply:
It looks like the intent here is different, introducing a new preference for the user. The feature flag implementation is designed around wanting to be prescriptive. It allows introducing new behavior and as it becomes stable push users to try it (while respecting their preferences). If there is any indication that the value of `default` might change, I'd suggest using the feature flag approach.